src/main/js/controls/Graph/Graph.js

```diff
@@ -454,6 +454,14 @@ class Graph extends Construct {
                 updateAxesDomain(this.config, this.content[position]);
             }
         }
+        if (graphData && this.config.showLabel) {
+            this.config.axis.x.label =
+                utils.sanitize(graphData.xlabel) || this.config.axis.x.label;
```

Comment:
It basically sanitizes the text by replacing the "<" and ">" symbols by "&lt" and "&gt". I sanitized the label before assigning to config. since I see it is being [done ](https://github.com/cerner/carbon-graphs/blob/f70b06952c0bcb2c32a560b8fe89a1c48b6663c9/src/main/js/controls/Graph/helpers/helpers.js#L535) while we create the label as well. Maybe d3 does not support it.